README.md

```diff
@@ -1,15 +1,170 @@
+</p>
 
-### Using the starter project
+# App Reporting dbt Package ([Docs](https://fivetran.github.io/dbt_github/))
+# 📣 What does this dbt package do?
+- Standardizes schemas from various app platform connectors and creates reporting models for all activity aggregated to the device, country, OS version, app version, traffic source and subscription levels. 
+- Currently supports the following Fivetran app platform connectors:
+    - [Apple App Store](https://github.com/fivetran/dbt_apple_store)
+    - [Google Play](https://github.com/fivetran/dbt_google_play)
+- Generates a comprehensive data dictionary of your source and modeled GitHub data via the [dbt docs site](https://fivetran.github.io/dbt_github/)
 
-Try running the following commands:
-- dbt run
-- dbt test
+Refer to the table below for a detailed view of final models materialized by default within this package. Additionally, check out our [Docs site](https://fivetran.github.io/dbt_app_reporting/#!/overview) for more details about these models. 
 
+| **model**                  | **description**                                                                                                                                               |
+| -------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| [app_reporting__app_version_report](https://dbt-github.netlify.app/#!/model/model.github.app_reporting__app_version_report)             | Each record represents daily metrics by app_name and app version.                                             |
+| [app_reporting__country_report](https://dbt-github.netlify.app/#!/model/model.github.app_reporting__country_report)     | Each record represents daily metrics by app_name and country |
+| [app_reporting__device_report](https://dbt-github.netlify.app/#!/model/model.github.app_reporting__device_report)     | Each record represents daily metrics by app_name and device.                             |
+| [app_reporting__os_version_report](https://dbt-github.netlify.app/#!/model/model.github.app_reporting__os_version_report)    | Each record represents daily metrics by app_name and OS version.                            |
+| [app_reporting__overview_report](https://dbt-github.netlify.app/#!/model/model.github.app_reporting__overview_report)   | Each record represents daily metrics by app_name.                            |
+| [app_reporting__subscription_report](https://dbt-github.netlify.app/#!/model/model.github.app_reporting__subscription_report)   | Each record represents daily metrics by app_name and subscription_name                            |
+| [app_reporting__traffic_source_report](https://dbt-github.netlify.app/#!/model/model.github.app_reporting__traffic_source_report) | Each record represents daily metrics by app_name and traffic source.                         |
```

Comment:
It looks like you have the netlify link instead of the Github pages link. This is something we will want to update in the GitHub readme. The form should look like this:
```
https://github.com/fivetran/dbt_app_reporting/blob/master/models/
```